### PR TITLE
Added some colors for warnings and errors

### PR DESCRIPTION
**Pull request by ellmetha**

### CODE REVIEW

**Comment by hsoft** [comment]
Looks good to me, but now it's your turn to review my fix.

**Comment by hsoft** [comment]
Oops, my fix was not well tested, it doesn't work. fixing...

**Comment by hsoft** [comment]
Now, that's better

File: nomad/project.py [hunk 1]
```diff
@@ -61,7 +61,8 @@ def shell(self, container_name=None):
             raise ProjectError(
                 'This action requires a container name to be specified because {count} '
                 'containers are defined in this project.'.format(count=len(self.containers)))
-        containers[0].shell()
+        for container in self._containers_generator(containers=containers):
```
Comment (ellmetha) [inline nomad/project.py]: Perhaps we could write a context manager to do this when we are considering a single container. It's a small improvement (not needed now I'd say), we could add it later. :wink:

Comment (hsoft) [inline nomad/project.py]: yeah, probably.

**Pull request merged**